src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -1983,6 +1983,7 @@ JvmtiHandshake::execute(JvmtiUnitedHandshakeClosure* hs_cl, jthread target) {
 
   jvmtiError err = JvmtiEnvBase::get_threadOop_and_JavaThread(tlh.list(), target, &java_thread, &thread_obj);
   if (err != JVMTI_ERROR_NONE) {
+    printf("DBG: JvmtiHandshake::execute: err: %d\n", (int)err); fflush(0);
```

Comment:
Any reason why not use UL instead with jvmti tag?

Reply:
Nice catch. The `printf` was temporarily used for debugging. Removed now.